locals.tf

```diff
@@ -0,0 +1,30 @@
+# ------------------------------------------------------------------------------
+# Retrieve the effective Account ID, User ID, and ARN in which Terraform is
+# authorized.  This is used to calculate the session names for assumed roles.
+# ------------------------------------------------------------------------------
+data "aws_caller_identity" "current" {}
```

Comment:
I strongly suggest naming this resource `default` instead of `current` for clarity.  I know we used to call it `current` in many places for some long-forgotten reason, but it is a terrible name.

Reply:
I'm assuming it's `current` because of it's use to populate `caller_user_name`? The idea being extract the user name of the current caller.

Reply:
Updated in ce45f34.